tests/effective_dps_test.cpp

```diff
@@ -529,7 +529,7 @@ static void check_knives( const std::function<Approx( const std::string & )> &ca
         CHECK( calc_expected_dps( "knife_hunting" ) == 11.0 );
         CHECK( calc_expected_dps( "kirpan_cheap" ) == 10.5 );
         CHECK( calc_expected_dps( "switchblade" ) == 10.5 );
-        CHECK( calc_expected_dps( "knife_meat_cleaver" ) == 14.5 );
+        CHECK( calc_expected_dps( "knife_meat_cleaver" ) == 17.22 );
```

Comment:
You should drop the damage or otherwise make adjustments, this is knife has no reason to have a DPS this high.

Reply:
Dropped the damage of the meat cleaver to match the vegetable cleaver at 4 bash and 15 cut